internals/cli/cmd_notify.go

```diff
@@ -47,6 +48,7 @@ func init() {
 		Description: cmdNotifyDescription,
 		ArgsHelp: map[string]string{
 			"--repeat-after": "Prevent notice with same type and key from reoccurring within this duration",
+			"--public":       "(admin only) Make the notice viewable by all users",
```

Comment:
Tweak suggestion (start with what the option actually does):

```suggestion
			"--public":       "Make the notice viewable by all users (admin only)",
```